src/main/Token.js

```diff
@@ -314,11 +314,12 @@ class Token {
     /**
      * Gets a list of available banks for linking
      *
+     * @param {Object} options - optional parameters for getBanks
      * @return {Promise} banks - list of banks
      */
-    getBanks() {
+    getBanks(options) {
```

Comment:
Why are we using options instead of passing each separately?

Reply:
too many params imo, you think it's better that way?

Reply:
No, just wondering the rationale, makes sense